packages/genesys-spark-tokens/data/$themes.json

```diff
@@ -1363,34 +1363,36 @@
       "background.interactive.danger.default": "71b66d28a4a34a562d03f8a9596b65739360de99",
       "background.interactive.danger.hover": "67e8649b1aa910a867c8a880e3189660a013ec17",
       "background.interactive.ghost.active": "5a63428c310eebf22faa6ee18f4c10812f5503b6",
+      "background.interactive.primary.default": "0188808b7c6d3225fc85570edd14072ada698fa2",
+      "background.interactive.primary.hover": "a285a137eb449419c1c85ba88c7ab91ef62bf9cf",
+      "background.interactive.primary.active": "6ef3e85d6ae0977961e8251602155649d5911bf7",
+      "background.interactive.secondary.default": "183a70ff8eafacbb8460480da8a7bd105264fd27",
+      "background.interactive.secondary.hover": "e975fd98a7d71b4744245f7189e53a82b7d8b175",
+      "background.interactive.secondary.active": "40ae7b1f1922e3474fab12043195ed2f3e282f7f",
+      "background.interactive.tertiary.default": "7bf179f212e34278d56e3f004662d173b14e9a15",
+      "background.interactive.tertiary.hover": "b79fb930972c72cdcdec61e8e5c8e88dc580cb08",
+      "background.interactive.tertiary.active": "c2f5bcd734300665a906c8a8f58286d793e9bf80",
```

Comment:
This does not look right.
I think these should be sorted alphabetically.
Are you sure the linter ran?